interop/hobbits/integration-tests.md

```diff
@@ -0,0 +1,19 @@
+# integration testing procedure
+
+![hit](hit.png)
+
+### Test 1: Coordinated Start
+
+1. Both clients are connected as above and started
+2. `Eth2Genesis` will be triggered immediately
+3. Messages between clients are routed through the relay
+4. The relay decodes and encodes each message as it passes thru the relay
+5. If the clients are able to justify and finalize, then they pass
+
+### Test 2: Join & Sync
+
+1. Both clients are connected as above and started
+2. Clients run for a couple of epochs
+3. A third client joins the network and asks to sync
```

Comment:
does third client contain validators or is it a non-validating node?

Reply:
i was think of a non-validating node initially

Reply:
makes sense. otherwise, you'd probably want to make sure 1st two clients have >2/3 of the vset